alchemy-web/docs/providers/cloudflare/custom-hostname.md

```diff
@@ -0,0 +1,187 @@
+# CustomHostname
+
+Create and manage Custom Hostnames for Cloudflare for SaaS. Custom Hostnames allow you to provide SSL certificates for your customer's domains that point to your Cloudflare zone.
```

Comment:
Link to https://developers.cloudflare.com/cloudflare-for-platforms/cloudflare-for-saas/domain-support/